templates/CRM/Contribute/Form/PaymentInfoBlock.tpl

```diff
@@ -53,6 +53,14 @@
    {else}
      {assign var='entity' value=$component}
    {/if}
-   {ts 1=$entity}No payments found for this %1 record{/ts}
+   <div class="odd-row">
+     {ts 1=$entity}&nbsp;&nbsp;&nbsp;No payments found for this %1 record.{/ts}
+     {if $isPending}
+       {capture assign=payNowLive}{crmURL p='civicrm/contact/view/contribution' q="reset=1&action=update&id=`$contribID`&cid=`$contactId`&context=`$context`&mode=live"}{/capture}
```

Comment:
yup it's just a cut paste from above https://github.com/civicrm/civicrm-core/pull/10776/files#diff-c901defeafe677e3a1c2e18b793d18bcL189